ansible/configs/sandbox/default_vars.yml

```diff
@@ -0,0 +1,177 @@
+---
+###### VARIABLES YOU SHOULD CONFIGURE FOR YOUR DEPLOYEMNT
+###### OR PASS as "-e" args to ansible-playbook command
+
+### Common Host settings
+# Repo Method. One of file, satellite and rhn
+repo_method: file
+
+#If using repo_method: satellite, you must set these values as well.
+# satellite_url: satellite.example.com
+# satellite_org: Sat_org_name
+# satellite_activationkey: "rhel7basic"
+
+# Do you want to run a full yum update
+update_packages: false
+
+## guid is the deployment unique identifier, it will be appended to all tags,
+## files and anything that identifies this environment from another "just like it"
+#guid: defaultguid
+
+# The next flag is 1 by default. If it is set to more than 1 then instead of creating
+# clientvm.guid.baseurl it will create clientvm{1..num_users}.guid.baseurl
+num_users: 1
+install_bastion: true
+install_common: true
+install_opentlc_integration: true
+install_ipa_client: false
+install_student_user: false
+install_ftl: false
+
+# install openshift python modules on bastion
+install_k8s_modules: true
+
+## Automatically deploy ocp4 cluster using the installer
+install_ocp4: false
+ocp4_installer_version: "4.1.0"
+cluster_name: "cluster-{{ guid }}"
```

Comment:
Thes variables are not relevant, right?

Reply:
updated

Reply:
and `install_k8s_modules` doesn't do anything